x-pack/test/functional/apps/dashboard/group3/reporting/screenshots.ts

```diff
@@ -210,26 +222,44 @@ export default function ({
         const reportData = await PageObjects.reporting.getRawReportData(url ?? '');
         sessionReportPath = await PageObjects.reporting.writeSessionReport(
           reportFileName,
-          'png',
+          'pdf',
           reportData,
           REPORTS_FOLDER
         );
         baselinePath = PageObjects.reporting.getBaselineReportPath(
           reportFileName,
-          'png',
+          'pdf',
           REPORTS_FOLDER
         );
-      });
-
-      after(async () => {
-        await esArchiver.unload('x-pack/test/functional/es_archives/reporting/ecommerce_76');
-        await kibanaServer.importExport.unload(
-          'x-pack/test/functional/fixtures/kbn_archiver/reporting/ecommerce_76.json'
-        );
+        expect(
+          await comparePdf(sessionReportPath, baselinePath, {
+            compareThreshold: 0.035,
+          })
+        ).to.be(true);
```

Comment:
I'm not satisfied that a test failure yields the message `Expected "false" to be "true"`, unfortunately it isn't the nature of `pdf-visual-compare` to yield an actual percentage of difference. That is something we do get with the compare PNG method. This PR should be reworked to use something more like an [older solution that was previously explored](https://github.com/elastic/kibana/pull/113910/files#diff-5eb27fd10caa675e771edbf53b0eb697c79977122ed75bb7fe8abc25d48467e6R64-R66).